its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java

```diff
@@ -102,6 +125,21 @@ public static Path getCustomRoslynPlugin() {
     return jars.get(0);
   }
 
+  public static TemporaryFolder createTempFolder() {
+    // If the test is being run under VSTS then the Scanner will
+    // expect the project to be under the VSTS sources directory
```

Comment:
Normally the temp folder will be created under the user folder, which means that when running under VSTS the scanner will complain that all of the source files are not under the root directory.